app/src/components/shared/Modal.jsx

```diff
@@ -70,7 +70,7 @@ Modal.propTypes = {
   /**
    * The callback method when closing the modal
    */
-  close: React.PropTypes.func.isRequired,
+  close: React.PropTypes.func,
```

Comment:
I disagree with that: when the user clicks the overlay or press ESC, the modal needs to call this callback in order to close itself. If we make it optional, we can forget to set the callback on other modals.

If you don't want the user to be able to close the modal, make the callback an empty function:
```javascript
< Modal close={() => {}} />
```